QUEST_LV_0200.tsv

```diff
@@ -9436,12 +9436,12 @@ QUEST_LV_0200_20151001_009439	I will be okay if I take little more rest.{nl}I am
 QUEST_LV_0200_20151001_009440	$Stogas Commanding Officer Ades
 QUEST_LV_0200_20151001_009441	You've brought the letter from Mesafasla?{nl}Then what happened to Roches?{nl}
 QUEST_LV_0200_20151001_009442	Ah... It's fortunate to hear that you are okay.
-QUEST_LV_0200_20151001_009443	You are a foreigner for a long time {nl}The supplies has been stopping from the kingdom, we don't know what's going on..{nl}
+QUEST_LV_0200_20151001_009443	$You have been a foreigner for a long time {nl}There has been no supply from the kingdom, we don't know what's going on..{nl}
```

Comment:
Period at the end of the first sentence.  Also might want to look at the original on this one, "You have been a foreigner for a long time" is kind of strange?